### ISSUE TITLE
Are you only use the position of two eys to determine if a person is sitting poorly？

### ISSUE THREAD

**Issue by sunbin1357** [monolesan/fix-posture#3]
Are you only use the position of two eyes to determine if a person is sitting poorly？

![image](https://user-images.githubusercontent.com/47880903/88913141-f70b6a00-d292-11ea-9810-c3ce754e0265.png)

**Comment by monolesan** [comment]
I use the position of only one eye.

**Comment by sunbin1357** [comment]
In general, your project is just using a pose estimation algorithm, and then use the position of  one eye to determine if a person is sitting poorly. Am I right in understanding this?

**Comment by monolesan** [comment]
Yes, you are right! Pretty simple, isn't it?

**Comment by sunbin1357** [comment]
Yes, very simple and effective. Have you recognize other actions using pose estimation algorithm, such as calling, wave, and so on.

**Comment by monolesan** [comment]
Yes, I have:
1. miroru (https://miroru.glitch.me/) —  3d head repeats your head rotation. Scroll down to read how it works.
2. make me big (https://make-me-big.glitch.me/) — font changes size based on your position (are you far or close) in front of screen.
